src/reset.css

```diff
@@ -46,10 +46,11 @@ a:not([class]) {
   text-decoration-skip-ink: auto;
 }
 
-/* Make images easier to work with */
+/* Make images and iframes easier to work with */
+iframe,
 img {
-  max-width: 100%;
   display: block;
+  max-width: 100%;
```

Comment:
Probably want to add an `img { height: auto }` in that case, else images may be squished.